src/LendingPool.sol

```diff
@@ -184,13 +183,13 @@ contract LendingPool is Owned, TrustedProtocol {
      * @param assets the amount of assets of the underlying ERC-20 token being withdrawn
      * @param receiver The address of the receiver of the underlying ERC-20 tokens
      */
-    function withdraw(uint256 assets, address receiver) public {
+    function withdrawFromLendingPool(uint256 assets, address receiver) public {
         _syncInterests();
 
-        require(supplyBalances[msg.sender] >= assets, "LP_W: Withdraw amount should be lower than the supplied balance");
+        require(redeemableAssetsOf[msg.sender] >= assets, "LP_W: Withdraw amount should be lower than the supplied balance");
```

Comment:
```suggestion
        require(redeemableAssetsOf[msg.sender] >= assets, "LP_W: Amount exceeds balance");
```

staying in one slot for the string

Reply:
In theory we could also skip this step since line 191 will catch it, or we can make line 191 unchecked